src/eval.c

```diff
@@ -1365,7 +1369,7 @@ char *ldbRespToHuman_Double(sds *o, char *reply) {
 /* Log a RESP reply as debugger output, in a human readable format.
  * If the resulting string is longer than 'len' plus a few more chars
  * used as prefix, it gets truncated. */
-void ldbLogRespReply(char *reply) {
```

Comment:
Why we need update Reps to Server?

Reply:
We didn't change RESP before. It is the name of the protocol.

We don't know what it means, It's just a name. Maybe now we can say it means Response Protocol. 😀

Reply:
There was a conflict at this line when I rebased. I didn't know which way to resolve, so resolved it to Server :) 
I have no preference either way. 

Do we want to refer to RESP in code base long-term or just remove references to it? eval.c still has many functions with Resp in it. 

```
char *ldbRespToHuman_Int(sds *o, char *reply);
char *ldbRespToHuman_Bulk(sds *o, char *reply);
char *ldbRespToHuman_Status(sds *o, char *reply);
char *ldbRespToHuman_MultiBulk(sds *o, char *reply);
char *ldbRespToHuman_Set(sds *o, char *reply);
char *ldbRespToHuman_Map(sds *o, char *reply);
char *ldbRespToHuman_Null(sds *o, char *reply);
char *ldbRespToHuman_Bool(sds *o, char *reply);
char *ldbRespToHuman_Double(sds *o, char *reply);
char *ldbRespToHuman(sds *o, char *reply) {
    case ':': p = ldbRespToHuman_Int(o, reply); break;
    case '$': p = ldbRespToHuman_Bulk(o, reply); break;
    case '+': p = ldbRespToHuman_Status(o, reply); break;
    case '-': p = ldbRespToHuman_Status(o, reply); break;
    case '*': p = ldbRespToHuman_MultiBulk(o, reply); break;
    case '~': p = ldbRespToHuman_Set(o, reply); break;
    case '%': p = ldbRespToHuman_Map(o, reply); break;
    case '_': p = ldbRespToHuman_Null(o, reply); break;
    case '#': p = ldbRespToHuman_Bool(o, reply); break;
    case ',': p = ldbRespToHuman_Double(o, reply); break;
/* The following functions are helpers for ldbRespToHuman(), each
char *ldbRespToHuman_Int(sds *o, char *reply) {
char *ldbRespToHuman_Bulk(sds *o, char *reply) {
char *ldbRespToHuman_Status(sds *o, char *reply) {
char *ldbRespToHuman_MultiBulk(sds *o, char *reply) {
        p = ldbRespToHuman(o, p);
char *ldbRespToHuman_Set(sds *o, char *reply) {
        p = ldbRespToHuman(o, p);
char *ldbRespToHuman_Map(sds *o, char *reply) {
        p = ldbRespToHuman(o, p);
        p = ldbRespToHuman(o, p);
char *ldbRespToHuman_Null(sds *o, char *reply) {
char *ldbRespToHuman_Bool(sds *o, char *reply) {
char *ldbRespToHuman_Double(sds *o, char *reply) {
    ldbRespToHuman(&log, reply);
```

Reply:
@zuiderkwast nevermind, I did not read your comment properly. I have my answer :)